milli/src/search/new/ranking_rules.rs

```diff
@@ -66,4 +67,6 @@ pub struct RankingRuleOutput<Q> {
     pub query: Q,
     /// The allowed candidates for the child ranking rule
     pub candidates: RoaringBitmap,
+    /// The score for the candidates of the current bucket
+    pub score: ScoreDetails,
```

Comment:
I assume you tested the score details feature with a placeholder search as well, and that it only returns the `Sort` score details? Also worth checking it when multiple `field:asc/desc` ranking rules exist.

Reply:
> I assume you tested the score details feature with a placeholder search as well

Yes, with placeholder search, only the ranking rules that are applied in the placeholder search have details, so basically only sort and geosort.

> Also worth checking it when multiple field:asc/desc ranking rules exist.

can't be sure if I already checked, can check again.

Reply:
Yeah, it works with multiple rules also.

example:

```json
{
      "title": "Batman: The Dark Knight Returns, Part 1",
      "id": 123025,
      "_rankingScoreDetails": {
        "words": {
          "order": 0,
          "matchingWords": 7,
          "maxMatchingWords": 7,
          "score": 1000
        },
        "typo": {
          "order": 1,
          "typoCount": 0,
          "maxTypoCount": 7,
          "score": 1000
        },
        "proximity": {
          "order": 2,
          "score": 1000
        },
        "attribute": {
          "order": 3,
          "attributes_ranking_order": 1000,
          "attributes_query_word_order": 761
        },
        "title:asc": {
          "order": 4,
          "value": "batman: the dark knight returns, part 1"
        },
        "release_date:desc": {
          "order": 5,
          "value": 1345507200.0
        },
        "exactness": {
          "order": 6,
          "exactIn": "matchesFull",
          "score": 1000
        }
      }
    }
```